src/troubleshooting.rst

```diff
@@ -0,0 +1,204 @@
+the ``cylc poll`` command (also available in the GUI).
+
+Use the ``cylc config`` command to inspect the platform's configuration to
+determine the configured polling schedule.
+
+Then check the ``job-activity.log`` file, there may have been a problem polling
+the remote platform, e.g. a network or configuration error.
+
+Likely causes:
+
+* The platform is down (e.g. all login nodes are offline).
+* There is a network issue.
+* Non-interactive SSH has not been correctly configured.
+
+
+My Job Submit-Failed
+^^^^^^^^^^^^^^^^^^^^
+
+A submit-failed job means one of three things:
+
+1. There is a Bash syntax error in the task configuration.
+
+   E.G. the following ``script`` has a syntax error, it is missing a
+   ``"`` character:
+   
+   .. code-block:: cylc
+   
+      [runtime]
+          [[foo]]
+              script = echo "Hello $WORLD
+   
+   This will result in a submission-failure which should appear in the
+   ``job-activity.log`` file (and also the scheduler log) something like this:
+   
+   .. code-block::
+   
+      /path/to/job.tmp: line 46: unexpected EOF while looking for matching `"'
+      /path/to/job.tmp: line 50: syntax error: unexpected end of file
+
+2. There was an error submitting the job to the specified platform (including
+   network issues).
+
+   See the ``job-activity.log`` and the scheduler log. The error should be in
+   one or both of those files.
+
+3. The platform is not correctly configured.
+
+
+My Job Failed
+^^^^^^^^^^^^^
+
+This means something went wrong executing the job.
+
+To find out more, see the ``job.err`` file.
+
+If you're struggling to track down the error, you might want to restart the
+workflow in debug mode and run the task again:
+
+.. code-block:: console
+
+   # shut the workflow down (leave any active jobs running)
+   $ cylc stop --now --now <workflow>
+   # restart the workflow in debug mode
+   $ cylc play <workflow> --debug
+   # re-run all failed task(s)
+   $ cylc trigger '<workflow>//*:failed'
+
+When a workflow is running in debug mode, all jobs will create a ``job.xtrace``
+file which can help you to locate the error within the job script.
+
+
+My workflow shutdown unexpectedly
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When a Cylc scheduler shuts down, it should leave behind a log message explaining why.
+
+E.G. this message means that a workflow shut down because it was told to:
+
+.. code-block::
+
+   Workflow shutting down - REQUEST(CLEAN)
+
+If a workflow shut down due to a critical problem, you should find some
+traceback in this log. If this traceback doesn't look like it comes from your
+system, please report it to the Cylc developers for investigation (on
+GitHub or Discourse).
+
+In some extreme cases, Cylc might not be able to write a log message e.g:
+
+* There's not enough disk space for Cylc to write a log message.
+* If the scheduler will killed in a nasty way e.g. ``kill -9``.
+* If the scheduler host goes down (e.g. power off).
```

Comment:
```suggestion
* If the scheduler host goes down (e.g. power off). 
  (Note if this happens you can simply play the workflow again once the host is back up - it will start up from where it left off when the host went down)
```

Reply:
This should be true for all cases, but probably isn't, so leaving it out for now.